src/common.rs

```diff
@@ -95,3 +95,8 @@ impl WriteConcern {
         bson
     }
 }
+
+pub fn merge_options<T: Into<bson::Document>>(document: bson::Document, options: T) -> bson::Document {
```

Comment:
Would this make more sense as a function on bson::Document? Feels reaal funny to have this global function, and languages like ruby have this type of method on the hash object itself. `doc.merge(other_doc)`